import numpy as np

<<<<<<< HEAD
<<<<<<< HEAD
np.little_endian = np.little_endian  # E: Cannot assign to final
=======
np.Inf = np.Inf  # E: Cannot assign to final
np.ALLOW_THREADS = np.ALLOW_THREADS  # E: Cannot assign to final
np.little_endian = np.little_endian  # E: Cannot assign to final
np.UFUNC_PYVALS_NAME = "bob"  # E: Incompatible types
np.CLIP = 2  # E: Incompatible types
>>>>>>> 596fb2e (Added incomplete requirements.txt)
=======
np.little_endian = np.little_endian  # E: Cannot assign to final
>>>>>>> 6d4acca (added stuff so I can run on nixos)
