from pathlib import Path
import numpy as np

path: Path
<<<<<<< HEAD
<<<<<<< HEAD
d1: np.lib.npyio.DataSource
=======
d1: np.DataSource
>>>>>>> 596fb2e (Added incomplete requirements.txt)
=======
d1: np.lib.npyio.DataSource
>>>>>>> 6d4acca (added stuff so I can run on nixos)

d1.abspath(path)  # E: incompatible type
d1.abspath(b"...")  # E: incompatible type

d1.exists(path)  # E: incompatible type
d1.exists(b"...")  # E: incompatible type

d1.open(path, "r")  # E: incompatible type
d1.open(b"...", encoding="utf8")  # E: incompatible type
d1.open(None, newline="/n")  # E: incompatible type
