import numpy as np

np.testing.bob  # E: Module has no attribute
np.bob  # E: Module has no attribute

# Stdlib modules in the namespace by accident
np.warnings  # E: Module has no attribute
np.sys  # E: Module has no attribute
<<<<<<< HEAD
<<<<<<< HEAD
np.os  # E: Module "numpy" does not explicitly export
=======
np.os  # E: Module has no attribute
>>>>>>> 596fb2e (Added incomplete requirements.txt)
=======
np.os  # E: Module "numpy" does not explicitly export
>>>>>>> 6d4acca (added stuff so I can run on nixos)
np.math  # E: Module has no attribute

# Public sub-modules that are not imported to their parent module by default;
# e.g. one must first execute `import numpy.lib.recfunctions`
np.lib.recfunctions  # E: Module has no attribute

<<<<<<< HEAD
<<<<<<< HEAD
=======
np.__NUMPY_SETUP__  # E: Module has no attribute
>>>>>>> 596fb2e (Added incomplete requirements.txt)
=======
>>>>>>> 6d4acca (added stuff so I can run on nixos)
np.__deprecated_attrs__  # E: Module has no attribute
np.__expired_functions__  # E: Module has no attribute
