import numpy as np

<<<<<<< HEAD
<<<<<<< HEAD
np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
=======
# Technically this works, but probably shouldn't. See
#
# https://github.com/numpy/numpy/issues/16366
#
np.maximum_sctype(1)  # E: No overload variant

np.issubsctype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type

np.find_common_type(np.int64, np.int64)  # E: incompatible type
>>>>>>> 596fb2e (Added incomplete requirements.txt)
=======
np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
>>>>>>> 6d4acca (added stuff so I can run on nixos)
